src/main/java/org/alliancegenome/curation_api/services/DiseaseAnnotationFmsService.java

```diff
@@ -295,7 +295,9 @@ private ExperimentalCondition validateExperimentalConditionFmsDTO(ExperimentalCo
         }
         if (dto.getConditionClassId() != null) {
             ZecoTerm term = zecoTermDAO.find(dto.getConditionClassId());
-            if (term == null) return null;
+            if (term == null || term.getSubsets().isEmpty() || !term.getSubsets().contains("ZECO_0000267")) {
```

Comment:
@markquintontulloch you may want to pull this out into the constants so it can be used in all three classes?

Reply:
@oblodgett agreed, and done